src/internet_identity/src/active_anchor_stats/activity_counter.rs

```diff
@@ -0,0 +1,9 @@
+use internet_identity_interface::internet_identity::types::Timestamp;
+
+pub mod active_anchor_counter;
+pub mod domain_active_anchor_counter;
+
+pub trait ActivityCounter: Clone {
```

Comment:
Maybe add some documentation, to explain the intended functionality of the trait?